src/resmom/linux/mom_mach.h

```diff
@@ -72,10 +72,11 @@ typedef unsigned long ulong;
 #if MOM_ALPS
 #include <sys/types.h>
 #include <dlfcn.h>
-#include "/usr/include/job.h"
```

Comment:
This seems like a specific include for Cray ALPS, it might not even be the pbs job.h file, why did you remove it?

Reply:
Ah, I assumed it was an erroneous inclusion. I'll add it back.